evaluate.py

```diff
@@ -2,27 +2,25 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import, division, print_function
 
-import itertools
 import json
 import sys
-
 from multiprocessing import cpu_count
 
 import absl.app
 import numpy as np
 import progressbar
 import tensorflow as tf
 import tensorflow.compat.v1 as tfv1
-
 from ds_ctcdecoder import ctc_beam_search_decoder_batch, Scorer
 from six.moves import zip
-
 from util.config import Config, initialize_globals
 from util.evaluate_tools import calculate_report
 from util.feeding import create_dataset
 from util.flags import create_flags, FLAGS
-from util.logging import log_error, log_progress, create_progressbar
-from util.helpers import check_ctcdecoder_version; check_ctcdecoder_version()
+from util.helpers import check_ctcdecoder_version
+from util.logging import create_progressbar, log_error, log_progress
+
+check_ctcdecoder_version()
```

Comment:
Please do not change the order and call to `check_ctcdecoder_version()`

Reply:
Sorry, this was automatic reordering. I undid it again.
But what is this order necessary for?

Reply:
It's the same in other parts of the code, and it makes more sense to tie them together.